nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/test/java/org/nuxeo/elasticsearch/test/io/TestDocumentModelReader.java

```diff
@@ -102,6 +103,7 @@ public void ICanReadADocModelFromJson() {
         assertFalse(doc.isFolder());
         assertFalse(doc.isVersion());
         assertFalse(doc.isLocked());
+        assertTrue(doc.hasFacet(COLD_STORAGE));
```

Comment:
This is not the responsibility of this test to assert that, otherwise this means we need to have this kind of assertion for all facets which is a lot of assertions + difficult to maintain for no real gain (I assume reading facet from json is tested elsewhere). I'm in favor of reverting it.